idempotency_key/locks.py

```diff
@@ -26,3 +29,20 @@ def acquire(self, *args, **kwargs) -> bool:
 
     def release(self):
         self.storage_lock.release()
+
+
+class MultiProcessLock(IdempotencyKeyLock):
+    """
+    Should be used if a lock is required across processes. Not that this class uses Redis in order to perform the lock.
+    """
+    storage_lock = StrictRedis().lock(
```

Comment:
Well spotted - just added this code in. For some reason, I assumed this would come from the cache settings but I've removed the dependency on django-redis and I'm using the Redis package now for the MultiProcessRedisLock class. You now specify the location using the LOCATION settings under LOCK.